package.json

```diff
@@ -1,18 +1,17 @@
 {
     "private": true,
     "dependencies": {
-        "async": "^2.6",
-        "critical": "^1.1",
-        "del": "^3.0",
-        "gulp": "^3.9",
-        "gulp-favicons": "^2.2",
-        "gulp-imagemin": "^4.0",
-        "gulp-responsive": "^2.8",
-        "gulp-rev-all": "^0.9",
-        "imagemin-mozjpeg": "^6.0",
-        "imagemin-optipng": "^5.2",
-        "imagemin-svgo": "^6.0",
-        "merge-stream": "^1.0.1",
-        "request": "^2.83"
+        "axios": "^1.4.0",
+        "critical": "^1.3.10",
+        "del": "^5.1.0",
+        "gulp": "^4.0.2",
+        "gulp-favicons": "^3.0.0",
+        "gulp-imagemin": "^7.1.0",
+        "gulp-responsive": "github:AidanMontare/gulp-responsive",
```

Comment:
Why have we had to move from https://www.npmjs.com/package/gulp-responsive?

Reply:
because it hasn't updated in the last four years and it's not compatible with the newer version of nodejs and Shars (https://github.com/lovell/sharp)

Reply:
We are using `gulp-responsive` but from a different fork that has fixed the mentioned issues